### PR TITLE
data: bL34cHig0

### PR DESCRIPTION
**Pull request by bL34cHig0**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/bL34cHig0.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Habeeb Kareem",
+  "bio": "Aspiring Pentester | IT graduate | Open-Source Enthusiast"
```
Comment (Dun-sin) [inline data/bL34cHig0.json:3]: You've got a problem here, either commit this change or push from your local environment

```suggestion
  "bio": "Aspiring Pentester | IT graduate | Open-Source Enthusiast",
```

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: data/bL34cHig0.json (hunk 1 above)
Comment (Viveksati5143) [inline data/bL34cHig0.json:3]: Yes, you should add a `,` in the end of line 3.

**Review by Viveksati5143 — CHANGES_REQUESTED** [review]

File: data/bL34cHig0.json (hunk 1 above)
Comment (bL34cHig0) [reply data/bL34cHig0.json:3]: oh yeah, thanks for pointing that out for me.

Comment (bL34cHig0) [reply data/bL34cHig0.json]: @Dun-sin, thanks. I committed the change.

**Review by Dun-sin — APPROVED** [review]
Looks good to me👐🏽

**Comment by bL34cHig0** [comment]
@Dun-sin appreciate it

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by bL34cHig0** [comment]
Appreciate it @SaraJaoude

**Pull request merged**